### ISSUE TITLE
D'Cent signing issue reported in rLogin

### ISSUE THREAD

**Issue by jessgusclark** [rsksmart/rLogin-providers#60]
See issue posted in rLogin here: https://github.com/rsksmart/rLogin/issues/284

**Comment by jessgusclark** [comment]
### Investigation, notes

I tested this issue out using our [rLogin Sample Dapp](https://basic-sample.rlogin.identity.rifos.org/) and it seems that the ethers.js and web3.js libraries is changing the message to hex before sending it to the device.

When I run this:

```ts
// rlogin.connect() stuff, then I get the provider:

rLogin.connect({ provider } => {
   const account = '0x0e6985...'
   provider.request({ method: 'personal_sign', params: ['Dapp Service', account ] })
}
```

I get this in my D'Cent wallet:

![Image from iOS](https://user-images.githubusercontent.com/766679/183078627-7a356eb0-be01-46fa-ae87-5a57eeab28c5.jpg)

However, when I use the web3.js or ethers.js library it sends the hex version to the device. You can see this in the console.logs:


![Screen Shot 2022-08-05 at 3 50 59 PM](https://user-images.githubusercontent.com/766679/183080915-8fc69cad-df3b-431b-b3e3-04f3179adb11.png)

The first request is via `provider.request` and the second one is via ethers.js library. 


## Trezor

I tested it with Trezor also and the same thing happens. Here is what is sent to the device from the sample app when I use the Ethers library:

![Screen Shot 2022-08-05 at 3 48 17 PM](https://user-images.githubusercontent.com/766679/183080444-25e69bcc-9a46-401a-a98b-37fb985b5933.png)

A hex value is sent to the device and the device converts it to a string. I suspect the same thing is happening on Ledger.

**Comment by jessgusclark** [comment]
I wonder if this is the issue:

https://github.com/rsksmart/rLogin-providers/blob/develop/packages/rlogin-dcent-provider/src/DCentProvider.ts#L73

The D'Cent provider is taking a hex value and converting it again to a hex value?

More research is needed here.

**Comment by jessgusclark** [comment]
Resolved with #61.
